src/main/java/mServer/crawler/sender/kika/tasks/KikaApiFilmTask.java

```diff
@@ -0,0 +1,235 @@
+  private static final Logger LOG = LogManager.getLogger(KikaApiFilmTask.class);
+
+  public KikaApiFilmTask(MediathekReader crawler, ConcurrentLinkedQueue<KikaApiFilmDto> urlToCrawlDTOs) {
+    super(crawler, urlToCrawlDTOs, Optional.empty());
+  }
+
+  @Override
+  protected JsonDeserializer<KikaApiVideoInfoDto> getParser(KikaApiFilmDto aDTO) {
+    return new KikaApiVideoInfoPageDeserializer();
+  }
+
+  @Override
+  protected Type getType() {
+    return new TypeToken<Set<KikaApiVideoInfoDto>>() {
+    }.getType();
+  }
+
+  @Override
+  protected void handleHttpError(KikaApiFilmDto dto, URI url, Response response) {
+    LOG.fatal(
+            "A HTTP error {} occurred when getting REST information from: \"{}\".",
+            response.getStatus(),
+            url);
+    Log.errorLog(324978334, String.format("A HTTP error %d occurred when getting REST information from: \"%s\".",
+            response.getStatus(),
+            url));
+  }
+
+  @Override
+  protected void postProcessing(KikaApiVideoInfoDto aResponseObj, KikaApiFilmDto aDTO) {
+    //
+    if (aResponseObj.getErrorCode().isPresent()) {
+      LOG.error("Error {} : {} for target {} ", aResponseObj.getErrorCode().get(), aResponseObj.getErrorMesssage().orElse(""), aDTO.getUrl());
+      Log.errorLog(324978335, String.format("Error %s} : %s for target %s ", aResponseObj.getErrorCode().get(), aResponseObj.getErrorMesssage().orElse(""), aDTO.getUrl()));
+      return;
+    }
+    //
+    if (aResponseObj.getVideoUrls().isEmpty()) {
+      LOG.error("No VideoUrls for {}", aDTO.getUrl());
+      return;
+    }
+    //
+    final Optional<LocalDateTime> airedDate = getAiredDateTime(aDTO);
+    if (!aDTO.getTitle().isPresent() || !aDTO.getTopic().isPresent() || !airedDate.isPresent() || !aDTO.getDuration().isPresent()) {
+      if (!aDTO.getTitle().isPresent()) {
+        LOG.error("Missing title for {}", aDTO.getUrl());
+      } else if (!aDTO.getTopic().isPresent()) {
+        LOG.error("Missing topic for {}", aDTO.getUrl());
+      } else if (!airedDate.isPresent()) {
+        LOG.error("Missing date for {}", aDTO.getUrl());
+      } else if (!aDTO.getDuration().isPresent()) {
+        LOG.error("Missing duration for {}", aDTO.getUrl());
+      }
+      return;
+    }
+    //
+    Map<Resolution, String> videoUrls = getVideoUrls(aResponseObj, aDTO);
+    Set<URL> subs = getSubtitle(aResponseObj, aDTO);
+
+    DatenFilm aFilm = new DatenFilm(
+            Const.KIKA,
+            "***" + aDTO.getTopic().get(),
+            aDTO.getWebsite().orElse(""),
+            aDTO.getTitle().get(),
+            videoUrls.get(Resolution.NORMAL),
+            ((subs.isEmpty()) ? "" : subs.toArray()[0].toString()),
+            airedDate.get().format(DateTimeFormatter.ofPattern("dd.MM.yyyy")),
+            airedDate.get().format(DateTimeFormatter.ofPattern("HH:mm:ss")),
+            parseDuration(aDTO, aDTO.getDuration()).get().getSeconds(),
+            aDTO.getDescription().orElse("")
```

Comment:
die neue Implementierung liefert nur den ersten Satz im Vergleich zur aktuellen. Wäre gut, wenn auch der Rest noch kommt.

Reply:
Ich habe jetzt erstmal keinen weiteren Text in den Quellen gefunden.
Die Text sind in der Webversion anders als in der App.
Daher kommt dann der Unterschied.

Reply:
ah ok, danke für die Erklärung. Dann passt das für mich.